src/frontend/src/optimizer/backfill_order_strategy.rs

```diff
@@ -35,7 +35,7 @@ use crate::session::SessionImpl;
 /// We only bind tables, materialized views and sources.
 /// Queries won't bind duplicate relations in the same query context.
 /// But backfill order strategy can have duplicate relations.
-pub fn bind_backfill_order_strategy(
+pub fn plan_backfill_order_strategy(
```

Comment:
Need to adjust the comments above?

Reply:
Will fix it in the downstream PR, otherwise leads to a lot of conflicts